march_hardware_interface/src/march_hardware_interface.cpp

```diff
@@ -205,6 +184,20 @@ void MarchHardwareInterface::init()
         }
       }
       joint.prepareActuation();
```

Comment:
Dit zet de IMC in de SWITCH_ON state?

Reply:
Yes, this function in turn calls the iMotionCube.goToOperationEnabled() function wherein we set the iMC to SWICTHED_ON and then to OPERATION_ENABLE.